opm/autodiff/BlackoilSolventModel_impl.hpp

```diff
@@ -284,6 +298,142 @@ namespace Opm {
         }
     }
 
+    template <class Grid>
+    void BlackoilSolventModel<Grid>::computeWellConnectionPressures(const SolutionState& state,
+                                                                        const WellState& xw)
+    {
+        if( ! Base::localWellsActive() ) return ;
```

Comment:
The differences are found in line 367 - 400, where the mixture of solvent and gas is accounted for in the calculations of  gas surface densities and bg. 

By construction the flow_solvent is mend to give the same results as flow if has_solvent_ is false. i.e. all differences should be found within if (has_solvent_) { }

I plan to restructure the code in order to reduce the code copying as much as possible in a separate PR.